server/server/api/views.py

```diff
@@ -10,7 +10,7 @@
 from common.arangodb import get_db
 
 from common.queries import CURRENCIES, DICTIONARIES, LANGUAGES, MENU, SUBMENU, PARAGRAPHS, PARALLELS, \
-    SUTTA_VIEW, SUTTAPLEX_LIST, IMAGES, EPIGRAPHS, WHY_WE_READ, DICTIONARYFULL, GLOSSARY
+    SUTTA_VIEW, SUTTAPLEX_LIST, IMAGES, EPIGRAPHS, WHY_WE_READ, DICTIONARYFULL, GLOSSARY, ADJACENT
```

Comment:
I'm not sure `ADJACENT` is descriptive enough as a resource name. You should be able to tell it has to do with dictionary terms just by looking at it after being woken up at 3 AM :). Don't worry about long names.